extension/src/test/suite/Formatter.test.ts

```diff
@@ -62,7 +62,7 @@ suite("Lisp Formatter mock Tests", function () {
   let internalLispFuncsStub;
   let internalOperators;
 
-  setup(() => {
+  setup(async () => {
```

Comment:
I think we should change "setup" to "suiteSetup" (like 'before' in BDD), and change "beforeEach" to "setup".
The code from line 65 to 75 should be done only once.

Reply:
No, the restore should be run for each case otherwise it will fail.

Reply:
Check the code. You're merging two methods into one, but I don't think all the code are needed to run before each test.

Reply:
Will update with the correct suitesetup.